src/tests/ftest/io/mdtest.yaml

```diff
@@ -0,0 +1,60 @@
+hosts:
+    test_servers:
+        - boro-A
+        - boro-B
+        - boro-C
+        - boro-D
+    test_clients:
+        - boro-E
+        - boro-F
+timeout: 600
+server_config:
+    name: daos_server
+pool:
+    createmode:
+        mode_RW:
+             mode: 146
+    createset:
+        setname: daos_server
+    createsize:
+        scm_size: 15000000000
+    createsvc:
+        svcn: 1
+mdtest:
+    client_processes:
+        np_8:
+            np: 8
+    api: 'DFS'
+    test_dir: "/"
+    iteration: 1
+    data: !mux
+        zero:
+            write_bytes: 0
+            read_bytes: 0
+        4K:
+            write_bytes: 4000
+            read_bytes: 4000
+    branch: !mux
+        zero:
+            branching_factor: 1
+            num_of_tasks: 10000
```

Comment:
im not sure what you mean here with number of tasks. is that the -n option?
in that case that should be the number of files and directories..

Reply:
10k for that is fine, but that also is different from your comment where you said that should be 100.. so im not sure TBH what you want with 10k here.

Reply:
Done for the first comment. Have changed num of file/dirs to reduce the overall run time. Will update the patch commit message as well.